python/pyarrow/array.pxi

```diff
@@ -2153,7 +2186,8 @@ cdef class StructArray(Array):
         return [pyarrow_wrap_array(arr) for arr in arrays]
 
     @staticmethod
-    def from_arrays(arrays, names=None, fields=None):
+    def from_arrays(arrays, names=None, fields=None, mask=None,
+                    memory_pool=None):
```

Comment:
It seems we are also inconsistent in the naming of this keyword, the `ListArray.from_arrays` above uses a `pool` keyword (but `memory_pool` is used more often, so this change is fine, will open a JIRA to make this consistent)

Reply:
-> https://issues.apache.org/jira/browse/ARROW-12805